File: src/main/java/kitchenpos/eatinorders/application/dto/CreateOrderRequest.java
package kitchenpos.eatinorders.application.dto;

import java.util.List;
import java.util.UUID;

public class CreateOrderRequest {
    private UUID orderTableId;
    private List<OrderLineItemDto> orderLineItems;

    public CreateOrderRequest() {
    }

    public CreateOrderRequest(UUID orderTableId, List<OrderLineItemDto> orderLineItems) {
        this.orderTableId = orderTableId;
        this.orderLineItems = orderLineItems;
    }

    public UUID getOrderTableId() {
        return orderTableId;
    }

    public List<OrderLineItemDto> getOrderLineItems() {
        return orderLineItems;
    }
}

File: src/main/kotlin/kitchenpos/tobe/eatinorder/application/dto/request/CreateOrderRequest.kt
package kitchenpos.tobe.eatinorder.application.dto.request

import java.math.BigDecimal
import java.util.UUID

data class CreateOrderRequest(
    val orderLineItems: List<OrderLineItem>,
    val orderTable: OrderTable,
) {
    data class OrderLineItem(
        val menuId: UUID,
        val quantity: Int,
        val price: BigDecimal,
    )

    data class OrderTable(
        val id: UUID,
        val name: String,
        val numberOfGuests: Int,
        val occupied: Boolean,
    )
}
